style.css

```diff
@@ -0,0 +1,1783 @@
+.img {
+  display: grid;
+  justify-content: center;
+  border-radius: 12px;
+}
+
+/* .main-img-d {
+  display: none;
+} */
+
+.img-1 {
+  display: block;
+  width: 295px;
+  height: 220px;
+  margin-left: 14px;
+}
+
+.img-2 {
+  display: block;
+
+  /* background-image: url(./Pictures/Snapshoot\ Portfolio-2.png); */
+  width: 295px;
+  height: 220px;
+  margin-left: 14px;
+}
+
+.img-3 {
+  display: block;
+
+  /* background-image: url(./Pictures/Snapshoot\ Portfolio-3.png); */
+  width: 295px;
+  height: 220px;
+  margin-left: 14px;
+}
+
+.img-4 {
+  display: block;
+
+  /* background-image: url(./Pictures/Snapshoot\ Portfolio-4.png); */
+  width: 295px;
+  height: 220px;
+  margin-left: 14px;
+}
+
+/* It is all the main portion that are heading, description, Frame, language button  */
+.card-main {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto;
+  gap: 20px;
+  padding: 0 12px;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* It show the heading design where grid column show that are 3 column */
+.card-name-h1 h1 {
+  color: #172840;
+  font-size: 32px;
+}
+
+/* Using the grid template area we are assign the 1 row and 3 columns */
+.experience ul {
+  display: grid;
+  grid-template-areas: "a b c";
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+/* So this that we are assign some variable  */
+.a {
+  grid-area: a;
+  list-style: none;
+  color: #344563;
+  font-size: 14px;
+}
+
+.b {
+  grid-area: b;
+  color: #7a869a;
+}
+
+.c {
+  grid-area: c;
+  color: #7a869a;
+}
+
+/* In this we are assign one row to it auto mean that jsut the space
+  accourding the the lenght of the paragraph */
+.parga {
+  display: grid;
+  grid-template-rows: auto;
+}
+
+.parga p {
+  font-size: 18px;
+  font-weight: 400;
+  line-height: 19px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+}
+
+/* This is also that we have dicuss above but that are 1 row and 6 column */
+.language ul {
+  list-style: none;
+  float: left;
+}
+
+.language-used {
+  float: left;
+  margin: 0 8px;
+  padding: 8px 12px;
+  background: #ebebff;
+  border-radius: 12px;
+}
+
+/* Now we are doing button sizing, when we click and hover what happen also are there */
+.button {
+  display: grid;
+  grid-template-columns: auto auto auto auto auto;
+}
+
+.but {
+  grid-column-start: 0;
+}
+
+.button button {
+  font-size: 17px;
+  font-weight: 500;
+  font-family: 'Poppins', sans-serif;
+  color: #396df2;
+  letter-spacing: 2px;
+  border-radius: 12px;
+  padding: 9px;
+  border-color: #396df2;
+  background-color: white;
+  transition: all 1s ease-in-out 2s;
+}
+
+.button button:hover {
+  background-color: #6070ff;
+  color: #fff;
+  box-shadow: 6px 4px 8px #6070ff;
+}
+
+.button button:active {
+  background-color: #2230d2;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.About-myself {
+  width: 407px;
+  height: 1125px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding: 8% 0;
+  border: 1px solid #e5e5e5;
+}
```

Comment:
- [ ] Its very nice design but there are some more thing to work on in order to match the Figma design as it is the core requirement of your project. Please make sure you followed typography ( font, size) and spacing (margin, padding). 
### Your design👇
![abotst](https://user-images.githubusercontent.com/66992901/200014755-9f004ed2-657c-4445-9aa6-a371a684afe2.PNG)
### Figma template👇
![abotfigma](https://user-images.githubusercontent.com/66992901/200014820-957f5edf-d8a4-41a7-97b5-ae59e1d6c47a.PNG)